formats.json

```diff
@@ -2141,6 +2323,440 @@
         "dxgiFormat": null,
         "mtlFormat": "MTLPixelFormatASTC_12x12_sRGB"
     },
+    {
+        "type": "ASTC",
+        "vkFormat": "VK_FORMAT_ASTC_12x12_SFLOAT_BLOCK_EXT",
+        "typeSize": 1,
+        "blockWidth": 12,
+        "blockHeight": 12,
+        "blockDepth": 1,
+        "glInternalFormat": "GL_COMPRESSED_RGBA_ASTC_12x12_KHR",
+        "glDesktopOnly": false,
+        "glFormat": null,
+        "glType": null,
+        "dxgiFormat": null,
+        "mtlFormat": null
+    },
+    {
+        "type": "ASTC",
+        "vkFormat": "VK_FORMAT_ASTC_3x3x3_UNORM_BLOCK_EXT",
```

Comment:
@lexaknyazev apologies if this is not the right place to ask -- but where are those 3D Vulkan enums defined (or proposed, at least)? I'm investigating 3D ASTC support in various APIs and the only places where I found the definitions are the `OES_texture_compression_astc` extension for GL and this PR for Vulkan. The current Vulkan spec doesn't have those enums and the only related issue I found is https://github.com/KhronosGroup/Vulkan-Docs/issues/926, but there isn't any mention about them either.

Reply:
They come from a draft Vulkan extension within Khronos, which will be released once there are 2 implementations. When that might happen, I don't know. The enum values will not change.

Reply:
Wonderful, thanks for clarifying that :)